stories/2023-02-01-Overwatch-League/OWL_story.ipynb

```diff
@@ -0,0 +1,3730 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.        team_order = owl21_reduced["map_winner"].value_counts().argsort().drop("draw")
Line #12.        map_order = owl21_reduced["map_name"].value_counts().argsort()
</pre><p> </p><p>May be wrong but I don't think you need <code>.argsort()</code>  on these lines, you only use the order of the index (which is sorted by <code>.value_counts()</code> by default). </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>You are correct.</p>